src/frontend/github-oauth.ts

```diff
@@ -59,8 +71,18 @@ export default (opts: OAuthOptions): GithubOAuth => {
 			state
 		} = req.query as Record<string, string>;
 
+		const traceLogsEnabled = await booleanFlag(BooleanFlags.TRACE_LOGGING, false);
+		const tracer = new Tracer(logger.child(opts), "callback", traceLogsEnabled)
+
+		const timestampBefore = req.session["timestamp_before_oauth"] as number;
```

Comment:
nit: You shouldn't use string literal accessors.  Pretty sure eslint will warn you about this.  You should be using a typed session variable which you can add to L46 in app.ts.

Reply:
also, after the session var isn't needed anymore, you should delete it using `delete req.session.timestamp_before_oauth;`